### DISCUSSION TITLE
Instance and Raid Lockout World Bosses

### DISCUSSION THREAD

**Discussion by Snektopus** [chromiecraft/chromiecraft#3429] [Suggestions]
My suggestion would be to make a Instance and a short (perhaps 3 day) raid lock out for the World Bosses we have and the one's coming up in the future.

-It is not helpful to purposes of testing if only a small minority of players ever see the content 
-World Bosses drop decent but not game unbalancing gear
-Only a small amount of players see the content, for the vast majority it is 'dead' content, and labour from devs for the benefit of 10 or 20 players server wide
-Instancing would remove the random nature of spawn rates, freeing up player time spent 'camping' spawn location to play or test the game in other ways.
-Would have a meta effect of making more lower player required Raids, which I believe would help the community
-'Instance' can be similar to phasing out but invisible walls around the 'battle area'?

**Comment by dennots** [comment]
How about you guys open all this content to the whole server by making it ID locked on whatever timer you see fit and just have the Bosses respawn every 5 or 10 minutes, so everyone can do them if they want to on their lockout, this would only create more PUGs for people to do when 25m raids are not happening and keep players engaged. 

The current system is toxic, just creates unnecessary division in the server and limits the content there is to do at lvl 60 after people are done with the dungeons. Zul Gurub patch is coming with 4 new bosses that only this same handful of people will experience. Open them up, revitalize the server by adding content for everyone.

And lets not start with the argument that it wouldn't be Blizzlike, that ship sailed a long time ago.
